src/s1reader/s1_reader.py

```diff
@@ -349,28 +394,47 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     # burst sensing starts in prototyping test data
     burst_interval = 2.758277
```

Comment:
I know this is not part of this PR, but I was wondering where did the magic number above came from? Should we read it from the metadata?

Reply:
I did not write this line, but I would guess it can be calculated from `azimuthTime` from the consecutive two bursts, so I think it can be readable from the product annotation .xml file.

Reply:
This number was computed in roughly the same way @seongsujeong described. The difference being the process described above was applied to many consecutive bursts and the average was taken.

Reply:
Right, but this can change with time or different instruments (e.g., Sentinel-1C/D)?

Reply:
@gshiroma S1C is a problem for 2023 when we have data?

Also the explanation of the number is provided in the comments immediately [above](https://github.com/opera-adt/s1-reader/blob/71e2c9e589f9e010b4952a574fcda83806561339/src/s1reader/s1_reader.py#L345-L349).

Reply:
Thank you for all the info, @LiangJYu . We just need to make sure that this code is robust globally and to potential changes to the instrument (not sure if that's possible). Hopefully it will not cause wrong burst IDs for specific burst locations due to cumulative time differences. And yes, it's something to keep in mind for S1C and S1D too. When S1C data is available, the project will probably ask us to test data from those sensors. These type of hidden details are dangerous when we don't remember about them anymore. If the S1C process runs, maybe we'll forget to check whether this `burst_interval` is the same for S1C.

That's why I asked if there was no change of reading it from the metadata? I guess you and @hfattahi are aware of that when this was written?